packages/kbn-std/src/index.ts

```diff
@@ -18,3 +18,11 @@ export { unset } from './unset';
 export { getFlattenedObject } from './get_flattened_object';
 export { ensureNoUnsafeProperties } from './ensure_no_unsafe_properties';
 export * from './rxjs_7';
+export {
+  map$,
+  mapWithLimit$,
+  asyncMap,
```

Comment:
nice, we can get rid of `bluebird.map` in the follow-up OR. IIRC it was the last piece that blocks us from `bluebird` removal 🎉